build.gradle

```diff
@@ -286,7 +286,7 @@ subprojects { subproject ->
         testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testImplementation group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
 
-        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.4'
+        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.6-CCD-3853'
```

Comment:
Should be updated with proper release number once hmcts/ccd-test-definitions#58 is released.

Reply:
Resolved in 5eaf89e162d49dc8207e4d20aad21b64aed21ea4